runtime/drivers/chromeDriver.js

```diff
@@ -10,8 +10,7 @@ const program = require('commander');
 const fs = require('fs');
 const path = require('path');
 const { Before } = require('@cucumber/cucumber');
-const { UtamWdioService } = require('wdio-utam-service');
-const utamConfig = require('../utam.config');
+const { filterQuietTags } = require('../.././cucumber.js');
```

Comment:
Change to 
const { filterQuietTags } = require('../../cucumber.js');

Reply:
done